job_templates/ci_job.xml.em

```diff
@@ -246,6 +251,8 @@ if "%CI_ROS2_REPOS_URL%" EQU "" (
   set "CI_ROS2_REPOS_URL=@default_repos_url"
 )
 set "CI_ARGS=%CI_ARGS% --repo-file-url %CI_ROS2_REPOS_URL%"
+if "%CI_ROS2_SUPPLEMENTAL_REPOS_URL%" NEQ "" (
+  set "CI_ARGS=%CI_ARGS% --supplemental-repo-file-url %CI_ROS2_SUPPLEMENTAL_REPOS_URL%"
```

Comment:
Yeah, there's one already there that's unmatched [current file version](https://github.com/ros2/ci/blob/3925e4174484561062c6ee5bc6f98ba82b5af9bd/job_templates/ci_job.xml.em#L256), [master file version](https://github.com/ros2/ci/blob/master/job_templates/ci_job.xml.em#L249).